segments/np_spotify_mac.script

```diff
@@ -11,9 +11,9 @@ if process_list contains "Spotify" then
 			set track_name to name of current track
 			set artist_name to artist of current track
 			#set album_name to album of current track
-			set trim_length to 40
+			set trim_length to 50
 			if player state is playing then
-				set now_playing to "► " & artist_name & " - " & track_name
+				set now_playing to " " & artist_name & " - " & track_name
```

Comment:
Let us keep the original one for consistency. The changed one does not render for me anyways.